packages/node/src/backend.ts

```diff
@@ -37,6 +37,9 @@ export interface NodeOptions extends Options {
   /** Sets the number of context lines for each frame when loading a file. */
   frameContextLines?: number;
 
+  /** The maximum number of frames to include in a stacktrace. The Node default is 10. */
```

Comment:
```suggestion
  /** The maximum number of frames to include in a stacktrace. The default is 10. */
```
Node's default value may be different from the SDK's. IMO, mentioning Node's default value is optional; but mentioning SDK's default value is required.

Reply:
My thinking was that, at least for the moment, there effectively _is_ no SDK default, or rather, the SDK default is to not change anything, leaving us with Node's default. I agree that if we ever were to change the SDK default, I would change that comment.